ethcore/src/verification/verification.rs

```diff
@@ -516,6 +522,16 @@ mod tests {
 			InvalidTimestamp(OutOfBounds { max: None, min: Some(parent.timestamp() + 1), found: header.timestamp() }));
 
 		header = good.clone();
+		header.set_timestamp(2450000000);
+		check_fail(basic_test(&create_test_block_with_data(&header, &good_transactions, &good_uncles), engine),
+				   InvalidTimestamp(OutOfBounds { max: Some(get_time().sec as u64 + 30), min: None, found: header.timestamp() }));
```

Comment:
This will make the test fail spuriously if the blocks check happens to hit the next second interval. Test should just check if this is an `InvalidTimestamp` error.